src/routes/(app)/c/[channel_name]/+page.server.ts

```diff
@@ -0,0 +1,26 @@
+import { getDb } from '$lib/server';
+import { getPublicChannelByName } from '$lib/server/services/channels';
+import { getPosts } from '$lib/server/services/content';
+import { error } from '@sveltejs/kit';
+import type { PageServerLoad } from './$types';
+
+export const load = (async ({ params: { channel_name }, locals }) => {
```

Comment:
Add `PageServerLoad` type annotation and remove `satisfies`. This has become the unofficial convention of the application.